cardano-api/gen/Gen/Cardano/Api.hs

```diff
@@ -83,19 +87,23 @@ genExUnits = do
     , Alonzo.exUnitsSteps = exUnitsSteps'
     }
 
-genCostModel :: Range Int -> Gen Text -> Gen Integer -> Gen Alonzo.CostModel
-genCostModel r gt gi = do
-  map' <- Gen.map r ((,) <$> gt <*> gi)
-  return $ Alonzo.CostModel map'
+genCostModels :: Gen Alonzo.CostModels
+genCostModels = do
+  CostModel cModel <- genCostModel
+  lang <- genLanguage
+  case Alonzo.mkCostModel lang cModel of
+    Left err -> panic . Text.pack $ "genCostModels: " <> err
```

Comment:
Are we sure this case never happens?

Reply:
We're not, it depends on `mkEvaluationContext` but: https://input-output-rnd.slack.com/archives/C21UF2WVC/p1649876509637199

It currently returns nothing so for now we just panic. When the plutus team gives more debugging info we can improve.